README.md

```diff
@@ -958,6 +959,65 @@ util.inherits(Stream, EventEmitter);
 
 [⬆ back to TOC](#table-of-contents)
 
+### Arrow functions
+
+* [Arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+  should be used in place where callback function expected:
+
+  > Explanation:
+  > * `this` keyword in arrow function binds to the upper scope's. It prevents usage invalid `this` or usage less
+  >    efficient `Function.prototype.bind` function.
+  > * The notation is shorter than function expression's.
+
+  **Good:**
+
+  ```js
+  [1, 2, 3].map(x => {
+      // ...
+  });
+  ```
+
+  **Bad:**
+
+  ```js
+  [1, 2, 3].map(function (x) {
+      // ...
+  });
+  ```
+
+* Parentheses around arrow function parameters can be ommited.
```

Comment:
I used to add parentheses for one parameter because of consistency but now I prefer to omit them for cleaner and more readable code :)